Options/WeakAurasOptions.lua

```diff
@@ -7126,13 +7126,21 @@ function WeakAuras.CreateFrame()
         if not(combinedText == "") then
           combinedText = combinedText.."\n\n";
         end
-        combinedText = combinedText.."--"..childId.."\n";
+
+        local formatString
```

Comment:
Well we can make sure that it's something that won't come up usually, something like λ or so